packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -159,7 +160,20 @@ export class SummarizerNode implements IRootSummarizerNode {
 			}
 		}
 
-		const result = await this.summarizeInternalFn(fullTree, true, telemetryContext);
+		const incrementalContext: IIncrementalContext = {
+			sequenceNumber: this.wipReferenceSequenceNumber ?? this._changeSequenceNumber,
+			previousSequenceNumber: this._latestSummary?.referenceSequenceNumber ?? 0,
+			parentPath: this._latestSummary?.fullPath.path ?? "",
```

Comment:
We should think what other info might be useful for DDSs. For example, info about last summary attempt (and sequence number), if it failed.